docs/ru/assemble_4.md

```diff
@@ -0,0 +1,336 @@
+    </center>
+
+3. Припаяйте силовые провода каждого BEC к контактным площадкам одного из регуляторов (**<font color=red>красный</font>** к **«+»**, **черный** к **«-»**).
+
+    <center>
+        <img src="../assets/4/10.png" width=300 class="zoom border">
+        <img src="../assets/4/12_1.png" width=300 class="zoom border">
+    </center>
+
+## Сопряжение приёмника и пульта
+
+1. Подключите радиоприемник к разъему от BEC 5В.
+2. Установите джампер на вход (B/VCC).
+3. Подключите АКБ.
+4. Светодиод на радиоприемнике должен мигать.
+
+    <center>
+        <img src="../assets/4/13.png" width=300 class="zoom border">
+        <img src="../assets/4/13_1.png" width=300 class="zoom border">
+    </center>
+
+5. Зажмите кнопку На пульте зажать кнопку *BIND KEY*.
+6. Включите пульт (перещелкните *POWER*, не отпуская *BIND KEY*).
+
+    <img src="../assets/4/radio/binding.png" class="zoom center">
+
+7. Ждите синхронизации.
+8. Отсоедините джампер.
+9. Светодиод на приемнике должен гореть непрерывно.
+
+Если пульт заблокирован, необходимо перевести все стики в начальное положение:
+
+1. Левый стик в **центральной нижней позиции**.
+2. Правый стик в **центре**.
+3. Переключатели A, B, C, D в положение **"от себя"**.
+
+    <img src="../assets/4/radio/base.png" alt="Нейтральное положение стиков пульта" class="zoom center">
+
+Убедитесь, что PPM в меню RX Setup отключен:
+
+1. Для входа в меню удерживайте нажатой кнопку "ОК".
+2. Кнопками Up/Down выбираем меню "System setup", кнопкой "ОК" подтвердите выбор.
+3. Выберите "RX Setup".
+4. Выберите "Output mode".
+5. Убедитесь, что в открывшемся меню выбран пункт "PWM".
+6. Чтобы сохранить настройки, удерживайте нажатой кнопку "Cancel".
+
+## Сборка
+
+1. Установите 4 стойки "папа-мама" 6 мм на PDB.
+2. Подключите шлейф питания к PDB.
+3. Установите поликарбонатную пластину на стойки и зафиксировать нейлоновыми гайками.
+
+    <center>
+        <img src="../assets/4/14.png" width=300 class="zoom border">
+        <img src="../assets/4/15.png" width=300 class="zoom border">
+        <img src="../assets/4/16.png" width=300 class="zoom border">
+    </center>
+
+## **Установка полётного контроллера**
+
+1. Установите полетный контроллер на пластину с помощью двухстороннего скотча.
+2. Стрелки на полетном контроллере и центральной деке должны быть направлены в одну сторону.
+3. Подключите шлейф питания PDB к разъему *"POWER"* полетного контроллера, закрутив его в "косичку" для взаимной фиксации проводов.
+
+    <center>
+        <img src="../assets/4/18.png" width=300 class="zoom border">
+        <img src="../assets/4/18_1.png" width=300 class="zoom border">
+    </center>
+
+4. Установите 4 алюминиевые стойки 40 мм с помощью винтов М3х10.
+
+    <center>
+        <img src="../assets/4/19.png" width=300 class="zoom border">
+        <img src="../assets/4/19_1.png" width=300 class="zoom border">
+    </center>
+
+5. Подключите сигнальные провода регуляторов к полетному контроллеру следующим образом:
```

Comment:
We need a nice, simple, clear picture showing **EXACTLY** where each signal wire should go. Something like a scheme or something. I get absolutely confused by that each and every time I reassemble my drone, and each and every time it's a major pain in the ass.